src/api/PermissionsApi.js

```diff
@@ -60,7 +60,7 @@ export function getAcl(aclKey :UUID[]) :Promise<*> {
 /**
  * `PATCH /permissions`
  *
- * Updates the Ace for a particular ACL Key, only if the user is the owner of the ACL Key.
+ * Updates the ACL for a particular ACL Key, only if the user is the owner of the ACL Key.
```

Comment:
I think this is actually correct, though confusing. Please revert.

Reply:
Done.